packages/codex-context/src/backend.ts

```
@@ -1,375 +1,403 @@
+				js?.choices?.[0]?.message?.content ||
+				js?.choices?.[0]?.text ||
+				"";
+			const ret: {
+				text: string;
+				usage?: {
+					prompt_tokens?: number;
+					completion_tokens?: number;
+					total_tokens?: number;
+				};
+				raw?: any;
+			} = { text, raw: js };
```

ESLint

[error] Variable should have an immutability of at least "ReadonlyDeep" (actual: "Mutable").

(functional/prefer-immutable-types)